app/models/app_remote_kill.rb

```diff
@@ -0,0 +1,65 @@
+class AppRemoteKill < ActiveRecord::Base
+  belongs_to :app
+  belongs_to :split
+
+  attribute :first_bad_version, :app_version
+  attribute :fixed_version, :app_version
```

Comment:
In the design doc, this was gonna be `last_bad_version` but I realized that `fixed_version` gives it semantic parity with a feature completion's `version` - when you ship the bug fix, you can drop in your own version number without looking up the last version number that was bad. Also, this fails safe in a multi-branch development strategy where 0.9 was the last bad version, 1.0 contains the fix, and then 0.9.1 comes out to address another bug and it still has the original bug and wants the remote kill to remain active (it's certainly possible to edit the fixed_version if you want to say that 0.9.1 is fixed too, but this will default to not enabling the feature until the version you declared safe).